assembl/static2/js/app/pages/idea.jsx

```diff
@@ -103,6 +104,17 @@ export const transformPosts = (edges, messageColumns, additionnalProps = {}) =>
     .sort(creationDateLastDescendantComparator);
 };
 
+// Function that counts the total number of posts in a Bright Mirror debate section
```

Comment:
if it is Bright Mirror specific, why is it in this file and not in `brightMirrorFiction.jsx` ?